addon/config/ember-try.js

```diff
@@ -43,7 +43,7 @@ const embroider = {
 
 module.exports = async function () {
   return {
-    useYarn: true,
+    usePnpm: true,
```

Comment:
IMO it's fine to bundle, maybe we could do this in separate PR. Which may slow down things considering this was idle for some time already.

Reply:
yeah, I'm with ya, it's just, it needs to happen anyway, and as as you state, this PR's been open for ages, I figured since both changes (move to single package monorepo, and pnpm) are pretty small, and internal changes, it's better to combine rather than configure ci.yml entirely differently twice for each of the changes, if split.